### ISSUE TITLE
Are the SSL training and downstream task datasets mutually independent?

### ISSUE THREAD

**Issue by rushin682** [Xiyue-Wang/TransPath#18]
Hello, 

Thanks for the paper and rigorous evaluation. I noticed that the SSL pre-training was done using TCGA and PAIP tissue samples and the corresponding downstream evaluation task of WSI classification included TCGA-NSCLC, TCGA-RCC datasets. 

I am curious whether both the SSL training and WSI classification datasets are mutually independent? Or is it fair to assume that SSL pretaining has already seen the TCGA-NSCLC, RCC samples ? I would like to hear your thoughts about it. 

Thanks once again for a great repo. I hope to hear from you soon.

**Comment by Xiyue-Wang** [comment]
Hi,
   Thank you very much. Because there is no label information, some articles (dsmil) do the same, it is certain that this is possible, but this is only to verify our experiment,other experiments are relatively independent.

Moreover,other groups have already been able to use our weights and obtain impressive results , which can
improve 20% on clinical private data. So don't worry, it should be effective

Thanks
Xiyue Wang

**Comment by rushin682** [comment]
Great! Thanks for the quick response. I appreciate the explanation and perspective. 😄
